.circleci/continue-config.yml

```diff
@@ -107,91 +115,51 @@ jobs:
+  primary:
     jobs:
+      - do_nothing:
+          name: No Op Placeholder Global
       - apply_module:
           name: Global IaC
           path: ./global
           <<: *tf_job_defaults
+          filters: pipeline.parameters.global-changed
+
+      - do_nothing:
+          name: No Op Placeholder EKS
+          requires:
+            - No Op Placeholder Global
       - apply_module:
-          name: <<matrix.context-suffix>> EKS IaC <<matrix.context-suffix>>
+          name: <<matrix.context-suffix>> EKS IaC
           matrix:
             parameters:
               context-suffix: ["NAMER","EMEA"]
           path: ./namer-eks
           <<: *tf_job_defaults
           requires:
             - Global IaC
+            - No Op Placeholder Global
           context:
             - CERA-INIT-<<matrix.context-suffix>>-NEW
-      - apply_module:
-          name: <<matrix.context-suffix>> Platform IaC <<matrix.context-suffix>>
-          path: ./namer-platforms
-          matrix:
-            parameters:
-              context-suffix: ["NAMER","EMEA"]
-          <<: *tf_job_defaults
-          requires:
-            - <<matrix.context-suffix>> EKS IaC <<matrix.context-suffix>>
-          context:
-            - CERA-INIT-<<matrix.context-suffix>>-NEW
+          filters: pipeline.parameters.iaas-changed or pipeline.parameters.global-changed
 
-  
-  include_eks:
-    when: 
-      and: 
-        - not: << pipeline.parameters.global-changed >>
-        - equal: [ true, <<pipeline.parameters.iaas-changed>> ]
-
-    jobs:
-      - apply_module:
-          name: <<matrix.context-suffix>> EKS IaC
-          path: ./namer-eks
-          <<: *tf_job_defaults
-          matrix:
-            parameters:
-              context-suffix: ["NAMER","EMEA"]
-          context:
-            - CERA-INIT-<<matrix.context-suffix>>-NEW
       - apply_module:
           name: <<matrix.context-suffix>> Platform IaC
           path: ./namer-platforms
-          <<: *tf_job_defaults
           matrix:
             parameters:
               context-suffix: ["NAMER","EMEA"]
+          <<: *tf_job_defaults
           requires:
             - <<matrix.context-suffix>> EKS IaC
+            - No Op Placeholder EKS
           context:
             - CERA-INIT-<<matrix.context-suffix>>-NEW
-
-  include_platform_layer:
-    when:
-      and: 
-       - not: << pipeline.parameters.global-changed >>
-       - not: <<pipeline.parameters.iaas-changed>>
-       - equal: [ true, <<pipeline.parameters.platforms-changed>> ]
-
-    jobs:
-      - apply_module:
-          name: <<matrix.context-suffix>> Platform IaC
-          path: ./namer-platforms
-          <<: *tf_job_defaults
-          matrix:
-            parameters:
-              context-suffix: ["NAMER","EMEA"]
-          context:
-            - CERA-INIT-<<matrix.context-suffix>>-NEW
-  
+          filters: pipeline.parameters.platforms-changed or pipeline.parameters.global-changed or pipeline.parameters.iaas-changed
```

Comment:
Just as an fyi, if you don't like long lines you can break them up. Newlines in the expression are treated as whitespace so the following works too:
```yaml
filters: |
  pipeline.parameters.platforms-changed
  or pipeline.parameters.global-changed
  or pipeline.parameters.iaas-changed
```